ext/sqlite3/sqlite3.c

```diff
@@ -942,27 +949,23 @@ PHP_METHOD(SQLite3, createFunction)
 	zend_long flags = 0;
 	db_obj = Z_SQLITE3_DB_P(object);
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "sf|ll", &sql_func, &sql_func_len, &fci, &fcc, &sql_func_num_args, &flags) == FAILURE) {
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "sF|ll", &sql_func, &sql_func_len, &fci, &fcc, &sql_func_num_args, &flags) == FAILURE) {
+		zend_release_fcall_info_cache(&fcc);
```

Comment:
FWIW you have legit test failures and I think it's because of this line (didn't check).
The reason I think that is because the parameter parsing can fail at sql_func, but then fcc won't be filled in and be uninitialized. Calling this on uninitialized data is bound to give problems.